dh/sidh/sike_test.go

```diff
@@ -567,7 +567,7 @@ func ExampleKEM() {
 	}
 	fmt.Printf("%t\n", bytes.Equal(ssE, ssD))
 
-	// Bob performs encapsulation with Allices's public key
+	// Bob performs encapsulation with Allice's public key
```

Comment:
```suggestion
	// Bob performs encapsulation with Alice's public key
```

Reply:
Addressed in https://github.com/cloudflare/circl/pull/456/commits/2d44d26eb1448fd15e372f1e7b2fdc4d4efb0bcb